app/controllers/miq_ae_tools_controller.rb

```diff
@@ -203,7 +203,7 @@ def upload
   def export_datastore
     filename = "datastore_" + format_timezone(Time.now, Time.zone, "fname") + ".zip"
     disable_client_cache
-    send_data(MiqAeDatastore.export,
+    send_data(MiqAeDatastore.export(User.current_tenant),
```

Comment:
I *think* you should just be able to use `current_tenant` here. @kbrock is that correct and/or is that the correct convention we want to be using? I know I just made a PR that made use of `current_tenant` in a controller and that was how I used it.